templates/layout/parts/head.html.twig

```diff
@@ -31,7 +31,7 @@
  #}
 
 <!DOCTYPE html>
-<html lang="{{ lang }}"
+<html lang="{{ lang ?? 'en' }}"
```

Comment:
Maybe changes in this files is not mandatory, see internal chan discussion about `front/login.php` page not being redirected.